docker/Dockerfile.hdfs

```diff
@@ -21,6 +21,12 @@ ENV HADOOP_COMMON_LIB_NATIVE_DIR $HADOOP_INSTALL/lib/native
 ENV HADOOP_OPTS "-Djava.library.path=$HADOOP_INSTALL/lib/native"
 ENV LD_LIBRARY_PATH $LD_LIBRARY_PATH:$JAVA_HOME/jre/lib/amd64/server
 
+#Set your nameserver info in the docker image
```

Comment:
These two lines aren't required for every setup, but seem reasonable as comments.

How about:
`echo 'nameserver <your-nameserver>' >> /etc/resolvconf/resolv.conf.d/head && resolvconf -u`